tfhe/src/core_crypto/entities/compressed_modulus_switched_glwe_ciphertext.rs

```diff
@@ -0,0 +1,277 @@
+use self::packed_integers::PackedIntegers;
+use crate::conformance::ParameterSetConformant;
+use crate::core_crypto::fft_impl::common::modulus_switch;
+use crate::core_crypto::prelude::*;
+
+/// An object to store a ciphertext in little memory
```

Comment:
"in little memory feels weird"

"with/using less memory" might be better

Reply:
Done